faq.md

```diff
@@ -107,9 +107,9 @@ as handwritten code. If you experience any weirdness, file a bug!
 
 <!-- References -->
 
-[`@Binds`]: http://google.github.io/dagger/api/latest/dagger/Binds.html
-[`@Provides`]: http://google.github.io/dagger/api/latest/dagger/Provides.html
-[`dagger.producers.monitoring`]: http://google.github.io/dagger/api/latest/dagger/producers/monitoring/package-summary.html
+[`@Binds`]: api/latest/dagger/Binds.html
```

Comment:
I'm not sure we should rely on the fact that the Javadoc's URLs are relative to the user documentation's.